### DISCUSSION TITLE
FrakenPHP 1000% CPU

### DISCUSSION THREAD

**Discussion by gponty** [php/frankenphp#586] [Q&A] [answered]
Hi,

Frankenphp is taking up 1000% of my CPU :) Has anyone else experienced this issue? I'm using Linux Mint.

![image](https://github.com/dunglas/frankenphp/assets/34276317/7b1a93c8-bf21-4496-9fa1-472ecea726bf)

Thank you

**Comment by withinboredom** [comment, marked as answer]
> Frankenphp is taking up 1000% of my CPU

There's a guy running around here claiming he can't get it to use more than 2 cores, and here you are using 10 cores. :D

> Has anyone else experienced this issue?

Each 100% is a full usage of a core, so if you have 24 cores, your "maximum percentage" will be 2,400%. Only you know if that is expected or not. Like are you doing load tests, running a bunch of scripts or what.

If you aren't doing anything at all, then there's a really good chance you have a worker script specified but the actual worker script does not exist, so it's in an infinite loop trying to start workers, so check the logs.

**Comment by gponty** [comment]
Ok found !

It's because of xdebug :  https://bugs.xdebug.org/view.php?id=2216https://bugs.xdebug.org/view.php?id=2216
